content/millennium/r4/financial/support/insurance-plan.md

```diff
@@ -0,0 +1,71 @@
+---
+title: InsurancePlan | R4 API
+---
+
+# InsurancePlan
+
+<%= beta_tag %>
+
+* TOC
+  {:toc}
+
+## Overview
+
+InsurancePlan describes a health insurance offering comprised of a list of covered benefits, costs associated with those benefits, and additional information about the offering, such as who it is owned and administered by, a coverage area contact information.
+
+The following fields are returned if valued:
+
+* [InsurancePlan id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.status){:target="_blank"}
+* [Type](https://hl7.org/fhir/r4/insurnaceplan-definitions.html#InsurancePlan.type){:target="_blank"}
+* [Name](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.name){:target="_blank"}
+* [Period](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.period){:target="_blank"}
+* [OwnedBy](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.ownedBy){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:insurance_plan, :r4) %>
+
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for InsurancePlans that meet supplied query parameters:
+
+    GET /InsurancePlan?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Parameters
+
+Name                  | Required? | Type          | Description
+-----------------------|-----------|---------------|-------------------------------------------------------
+`owned-by`                 | Yes       | [`reference`]     | Organization associated with the InsurancePlan.
+
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir.applications.stagingcerner.com/r4/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/InsurancePlan?owned-by=589743
```

Comment:
Should the url be a staging url?

Reply:
It Should be a prod URL but since we do not have it in prod yet what would you suggest @BhavyaAcharya

Reply:
We can have Sandbox data for now, and update the examples once released to prod

Reply:
Can you please update it to open endpoint, like https://fhir-open.cerner.com/ ..

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/b15a0dd161ba774ee4a02ba6c8fc2d398615a7f3#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR52